.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+ci/* @TykTechnologies/devops
```

Comment:
Should this be `/ci/ @TykTechnologies/devops` ?

Ref:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

```

> # In this example, @doctocat owns any file in the `/docs`  
> # directory in the root of your repository and any of its  
> # subdirectories.  
> docs/ @doctocat

```

Reply:
Good catch @asutosh , thanks!!!